eng/Versions.props

```diff
@@ -60,16 +60,16 @@
   </ItemGroup>
   <PropertyGroup>
     <!-- Arcade dependencies -->
-    <MicrosoftDotNetApiCompatVersion>5.0.0-beta.20162.4</MicrosoftDotNetApiCompatVersion>
-    <MicrosoftDotNetBuildTasksFeedVersion>5.0.0-beta.20160.4</MicrosoftDotNetBuildTasksFeedVersion>
-    <MicrosoftDotNetCodeAnalysisVersion>5.0.0-beta.20160.4</MicrosoftDotNetCodeAnalysisVersion>
-    <MicrosoftDotNetGenAPIVersion>5.0.0-beta.20160.4</MicrosoftDotNetGenAPIVersion>
-    <MicrosoftDotNetGenFacadesVersion>5.0.0-beta.20160.4</MicrosoftDotNetGenFacadesVersion>
-    <MicrosoftDotNetXUnitExtensionsVersion>5.0.0-beta.20160.4</MicrosoftDotNetXUnitExtensionsVersion>
-    <MicrosoftDotNetXUnitConsoleRunnerVersion>2.5.1-beta.20160.4</MicrosoftDotNetXUnitConsoleRunnerVersion>
-    <MicrosoftDotNetBuildTasksPackagingVersion>5.0.0-beta.20160.4</MicrosoftDotNetBuildTasksPackagingVersion>
-    <MicrosoftDotNetRemoteExecutorVersion>5.0.0-beta.20160.4</MicrosoftDotNetRemoteExecutorVersion>
-    <MicrosoftDotNetVersionToolsTasksVersion>5.0.0-beta.20160.4</MicrosoftDotNetVersionToolsTasksVersion>
+    <MicrosoftDotNetApiCompatVersion>5.0.0-beta.20162.3</MicrosoftDotNetApiCompatVersion>
```

Comment:
Actually, it looks like this is downgrading the APIComat version? 😖

Reply:
Yeah, just noticed that as well. It seems like maybe the .4 hasn't made it to the Latest channel and it is still in Validation. Will remove that change in a sec.

Reply:
Weird - the version in `details.xml` was `5.0.0-beta.20160.4`, not `5.0.0-beta.20162.4`

Reply:
It seems like it was because we updated it manually but we didn't update details.xml

https://github.com/dotnet/runtime/commit/4480bdbe66c55caceb64fc3477010435ba70c4ec#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR63